src/plays/sticky-notes/App.js

```diff
@@ -31,16 +31,18 @@ function App() {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    const newNote = {
-      ...note,
-      id: Math.random() * 10
-    };
-
-    setNotes([newNote, ...notes]);
-    setNote({
-      body: ''
-    });
-    setToggleForm(!toggleForm);
+    if (note.body) {
```

Comment:
I think you need to pull your branch because this change is already part of the main.

Reply:
@atapas fixed the bugs. check it out.